src/modules/workspaces/workspaces.controller.spec.ts

```diff
@@ -234,11 +234,61 @@ describe('Workspace Controller', () => {
         mockResolvedValues,
       );
 
-      const getMembers = await workspacesController.getWorkspaceMembers(
+      await expect(
+        workspacesController.getWorkspaceMembers(workspace.id, owner),
+      ).resolves.toEqual(mockResolvedValues);
+
+      expect(workspacesUsecases.getWorkspaceMembers).toHaveBeenCalledWith(
+        workspace.id,
+        owner,
+        null,
+      );
+    });
+
+    it('When members are requested with "workspaceId" and "user", and additionally include a "search" then it should return workspaces members data according to the "search"', async () => {
```

Comment:
_When a search param is provided, then it should be used for querying workspace members_

Sounds easier and more concise